_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -22,16 +22,16 @@ For more information see the [RFC](https://community.theforeman.org/t/drop-requi
 
 Foreman 2.1 introduced EL8 support and Katello 4.0 (on Foreman 2.4) followed.
 While it's currently undecided when EL7 support will be dropped exactly, this is an early notice given the considerable number of EL7 deployments.
-For fresh installations, EL8 should considered the preferred target.
+For fresh installations, it is advisable to install on EL8.
 Existing installations should start thinking about a migration plan.
 
-Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
+Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

Comment:
I use host/client interchangeably. I found system unclear so I tried to clarify. I'd rather leave the "using Foreman" part and stick with hosts or host systems.

Reply:
A definition list in the planning guide over on docs.theforeman.org would be great. I've tried to curb random vague usage. I held up the publication of ATIX's Foreman ACD guide for days trying to get "service" taken out but I had no better offerings for them and no one else had either, so I had to let it pass. I'd prefer neologisms that exist only for Foreman and clearly defined than "Service/system/host" all used to mean something similar.

Reply:
I completely agree. It's confusing for users to read one thing in the docs and then see another thing in the UI. It may even be its own guide that all other guides could refer to.

On a side note, it's what I love about the name Katello: it's unique and there's no overlap with other existing names. Foreman has the problem that besides being a generic English word, it also has another Ruby project with the same name but serving a different goal. Worse, in development foreman.git actually lists the other foreman project as a development dependency.

Reply:
Pulp is even worse. A nightmare actually. 
There's the British band and several software projects all competing on Google
https://twitter.com/pulp_platform/
https://pypi.org/project/PuLP/
The YouTube auto close captioning on one of the engineer's demos picked up Poop instead of Pulp. That would actually be easier and more unique software project name =)